RecodoDesktop/RecodoDesktop/AuthorizationWindow.xaml.cs

```diff
@@ -31,15 +34,19 @@ private async Task GetToken(string endpoint)
             var auth = new AuthorizationService(endpoint);
             try
             {
-                var authResult = await auth.Authorize();
+                token = await auth.Authorize();
                 this.ProgressPanel.Visibility = Visibility.Hidden;
             }
             catch
             {
                 this.DeterminateCircularProgress.Visibility = Visibility.Hidden;
                 this.BrowserState.Text = "Some went wrong, please try again..";
             }
-            this.Activate();
+
+            this.Hide();
+            RecorderService recorderService = new RecorderService(token);
```

Comment:
One of two:
var recorderService = new RecorderService(token);
RecorderService recorderService = new(token);